docs/src/containers/WidgetForm.tsx

```diff
@@ -0,0 +1,210 @@
+import {
+  Button,
+  Dialog,
+  IconButton,
+  RadioButton,
+  RadioGroup,
+  TextField,
+} from "@statflo/ui";
+import { Formik } from "formik";
+import { useEffect, useState } from "react";
+import * as Yup from "yup";
+import { faker } from "@faker-js/faker";
+
+import { useWidgetContext } from "../providers/WidgetProvider";
+
+const WidgetSchema = Yup.object().shape({
```

Comment:
I haven't heard of yup. Is it a form builder?

Reply:
Or validation?

Reply:
It's a schema builder, which I am using for validating the form. It's what is recommended by `Formik` which is the form package that I am using.

Reply:
Cool. I'm a fan of form builders, I was wondering if there was any motivation to use on our react projects.